examples/fleet/kuard/README.md

```diff
@@ -0,0 +1,184 @@
+
+   ```shell
+   kubectl get crp kuard -o yaml | grep status -A 1000
+   ```
+
+   The output is similar to:
+
+   ```yaml
+   status:
+   conditions:
+     - lastTransitionTime: "2022-09-20T06:31:57Z"
+       message: Successfully scheduled resources for placement
+       observedGeneration: 1
+       reason: ScheduleSucceeded
+       status: "True"
+       type: Scheduled
+     - lastTransitionTime: "2022-09-20T06:31:58Z"
+       message: Successfully applied resources to member clusters
+       observedGeneration: 1
+       reason: ApplySucceeded
+       status: "True"
+       type: Applied
+   selectedResources:
+     - group: networking.fleet.azure.com
+       kind: ServiceExport
+       name: kuard
+       namespace: kuard-demo
+       version: v1
+     - kind: Service
+       name: kuard
+       namespace: kuard-demo
+       version: v1
+     - kind: Namespace
+       name: kuard-demo
+       version: v1
+   targetClusters:
+     - aks-member-3
+     - aks-member-2
+     - aks-member-1
+   ```
+   
+    All the resources under namespace `kuard-demo` (including the namespace) are selected and placed to all the member clusters.
+
+3. Switch to the member clusters to verify the resource has been created:
+
+   ```shell
+   KUBECONFIG=member1 kubectl get all -n kuard-demo
+   KUBECONFIG=member2 kubectl get all -n kuard-demo
+   KUBECONFIG=member3 kubectl get all -n kuard-demo
+   ```
+
+   The output is similar to:
+
+   ```console
+   NAME                        READY   STATUS    RESTARTS   AGE                  
+   pod/kuard-7788d9bc5-bp8qw   1/1     Running   0          13m                  
+   pod/kuard-7788d9bc5-tp54d   1/1     Running   0          13m
+
+   NAME            TYPE           CLUSTER-IP   EXTERNAL-IP   PORT(S)          AGE
+   service/kuard   LoadBalancer   10.0.188.8   10.2.0.62     8080:32629/TCP   13m
+
+   NAME                    READY   UP-TO-DATE   AVAILABLE   AGE                  
+   deployment.apps/kuard   2/2     2            2           13m
+
+   NAME                              DESIRED   CURRENT   READY   AGE             
+   replicaset.apps/kuard-7788d9bc5   2         2         2       13m
+   ```
+4. Switch to the member clusters to verify the service export has been created:
+   
+   ```shell
+   kubectl get serviceexport -n kuard-demo
+   ```
+
+   The output is similar to:
+   ```console
+   NAME    IS-VALID   IS-CONFLICTED   AGE
+   kuard   True       False           13m
+   ```
+   The service is valid for export (`IS-VALID` field is true) and has no conflicts with other exports (`IS-CONFLICT` is false).
+
+### 3. Expose fleet-wide endpoints from exported services with a multi-cluster service
+
+1. Switch to `aks-member-1` context to create a multi-cluster service. The endpoints will then be exposed with a L4 load balancer using `aks-member-1` nodes.
+
+   ```shell
+   kubectl apply -f https://raw.githubusercontent.com/Azure/AKS/master/examples/fleet/kuard/kuard-mcs.yaml

```

Comment:
KUBECONFIG=member1 kubectl apply -f https://raw.githubusercontent.com/Azure/AKS/master/examples/fleet/kuard/kuard-mcs.yaml

Reply:
It is pretty difficult to remember to switch between hub and member. Use commands like this will make it more explicit regarding which clusters we are working on.